src/backend/timeseries/sql/udfs/create_missing_partitions/10.2-1.sql

```diff
@@ -0,0 +1,35 @@
+CREATE OR REPLACE FUNCTION pg_catalog.create_missing_partitions(
+    table_name regclass,
+    to_date timestamptz,
+    start_from timestamptz DEFAULT NULL)
+returns boolean
+LANGUAGE plpgsql
+AS $$
+DECLARE
+    missing_partition_record record;
+BEGIN
+
+    /*
+     * Get missing partition range info using the get_missing_partition_ranges
+     * and create partitions using that info. Since timeseries table related checks are
+     * handled by get_missing_partition_range we don't check them here again.
+     *
+     * TODO: Create that using attach partition, which can be implemented with another UDF to make sure that it gets right locks.
+     * Consider different types of tables while implemented that (Task 1.3 on sheet)
+     */
+
+    FOR missing_partition_record IN
+        SELECT *
+        FROM get_missing_partition_ranges(table_name, to_date, start_from)
+    LOOP
+        EXECUTE format('CREATE TABLE %I PARTITION OF %I FOR VALUES FROM (''%I'') TO (''%I'')', missing_partition_record.partition_name, table_name::text, missing_partition_record.range_from_value, missing_partition_record.range_to_value);
```

Comment:
are the names guaranteed to be fully-qualified?

Reply:
If they are, be careful that %I will blindly put quotes around fully-qualified names.
If they are not, be careful about tables that fall outside the search_path.